### PR TITLE
MTM-46296 update the docs with the info about mutlti-type filtering in notification2 subscriptions

### PR DESCRIPTION
**Pull request by sag-tjegorow**

### CODE REVIEW

File: content/reference/notifications-bundle/overview.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Subscriptions are set up using the [subscription method](https://{{<domain-c8y>}
 This API requires the calling user to be an authenticated {{< product-c8y-iot >}} user and to have the new ROLE_NOTIFICATION_2_ADMIN role.
 
 When subscribing to notifications, a filter for notifications can be specified which determines the APIs (alarms, alarms with children, events, events with children, measurements, managed objects, operations or any combination of these) to filter by. The alarms with children and events with children enable users to create explicit subscriptions that allow the delivery of child as well as parent managed object events and alarms.
-It is also possible to filter by presence of a specific JSON fragment or "fragment type".
+It is also possible to filter by presence of a specific value of a JSON "type" attribute, either by direct comparison, or by OData-like expression when it is defined (at the moment only `or` operator is supported which allows filtering of multiple types with the use of an expression `'type1' or 'type2' [or 'typeN']`)
```
Comment (BeateRixen) [inline content/reference/notifications-bundle/overview.md:30]:
```suggestion
It is also possible to filter by the presence of a specific value of a JSON "type" attribute, either by direct comparison, or by an OData-like expression when it is defined (at the moment only the "or" operator is supported which allows filtering of multiple types with the use of an expression `'type1' or 'type2' [or 'typeN']`).
```

**Review by slc-softwareag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@sag-tjegorow Shall I merge this? This goes into 10.16+. To which older releases should this be grafted?

**Comment by sag-tjegorow** [comment]
> @sag-tjegorow Shall I merge this? This goes into 10.16+. To which older releases should this be grafted?

Yes, this will go to 10.16+ and it won't be grafted. But I need to update the documentation first because we've changed the implementation a bit. I'll post another comment when it's done.

**Review by andrekramer1 — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Review by rohitjoshi-softwareag — APPROVED** [review]

**Comment by sag-tjegorow** [comment]
> > @sag-tjegorow Shall I merge this? This goes into 10.16+. To which older releases should this be grafted?
> 
> Yes, this will go to 10.16+ and it won't be grafted. But I need to update the documentation first because we've changed the implementation a bit. I'll post another comment when it's done.

@BeateRixen the documentation is already updated, and the change in core was merged last Friday so you can merge it to 10.16 (there are no grafts).

**Pull request merged**